<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>85-try and catch</title>
</head>
<body>

<button id="try">Cliquez sur le bouton</button>

<script
        src="https://code.jquery.com/jquery-3.4.1.min.js"
        integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
        crossorigin="anonymous"></script>
<script>
    $(function () {
        $('#try').click(function () {
            var message='';
            try {
                alort('un message');
            }
            catch(err) {
                message = 'Une erreur s\'est produite\n\n';
                message += 'Description : ' +err.message + '\n\n';
                message += 'Cliquez sur Ok pour continuer';
                alert(message);

                // function gestionError(err) {
                //     alert('Erreur : \n' + err);
                //     return true;
                // }
                // window.onerror = gestionError();
            }
        });
    })
</script>
</body>
</html>